Gemfile.lock

```diff
@@ -1,26 +1,26 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.6)
+    activesupport (5.1.3)
```

Comment:
Hi Vaishali, we haven't changed the gemfile. So we are clueless as to how it got changed. Can you tell us what changes need to be made?